sklearn/utils/_estimator_html_repr.py

```diff
@@ -530,6 +544,4 @@ def _get_doc_link(self):
             return self._doc_link_template.format(
                 estimator_module=estimator_module, estimator_name=estimator_name
             )
-        return self._doc_link_template.format(
-            **self._doc_link_url_param_generator(self)
-        )
+        return self._doc_link_template.format(**self._doc_link_url_param_generator())
```

Comment:
So here, we don't need `self` since one should use `types` to bind the callable as a class method.